TPL/force-app/main/default/flows/Update_Health_Care_Cost_Fields_On_Create.flow-meta.xml

```diff
@@ -188,6 +188,13 @@
                 <elementReference>LoopThroughHealthCareCost.Id</elementReference>
             </value>
         </assignmentItems>
+        <assignmentItems>
```

Comment:
This is the actual assignment, which is done universally on all healthcare cost records.